sample.env

```diff
@@ -0,0 +1,7 @@
+MONGOHQ_URL=
+FB_CLIENT_ID=
+FB_CLIENT_SECRET=
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
```

Comment:
It's either you give env keys values or don't put them at all. What exactly does this file do?